Campaign/app/Http/Controllers/CampaignController.php

```diff
@@ -105,13 +105,22 @@ public function json(Datatables $dataTables, SegmentAggregator $segmentAggregato
             })
             ->addColumn('segments', function (Campaign $campaign) use ($segments) {
                 $segmentNames = [];
+
+                $exclusiveIcon = '<i class="zmdi zmdi-eye-off"></i>';
+                $inclusiveIcon = '<i class="zmdi zmdi-check primary-color"></i>';
+
                 foreach ($campaign->segments as $segment) {
+                    $icon = $segment->inclusive ? $inclusiveIcon : $exclusiveIcon;
+
                     if ($segments->get($segment->code)) {
-                        $segmentNames[] = "- <span title='{$segment->code}'>{$segments->get($segment->code)}</span></em>";
+                        $segmentNames[] = "{$icon} <span title='{$segment->code}'>{$segments->get($segment->code)}</span></em>";
                     } else {
-                        $segmentNames[] = "- <span title='{$segment->code}'>{$segment->code}</span></em>";
+                        $segmentNames[] = "{$icon} <span title='{$segment->code}'>{$segment->code}</span></em>";
                     }
                 }
+
+                sort($segmentNames, SORT_STRING);
```

Comment:
The order of segments is important during evaluation. It'd be good to keep the sorting of segments "as is" even here so users of the system know, in which order the segments will get evaluated.

Therefore I wouldn't sort the names here.

Reply:
Handled, it'll get here once internal repo will get mirrored here.